radixdlt/src/main/java/com/radixdlt/consensus/validators/ValidatorSet.java

```diff
@@ -57,7 +70,24 @@ public static ValidatorSet from(Collection<Validator> validators) {
 	 * @return An initial validation state with no signatures
 	 */
 	public ValidationState newValidationState(Hash anchor) {
-		return ValidationState.forValidatorSet(anchor, validators.keySet());
+		return ValidationState.forValidatorSet(anchor, this);
+	}
+
+	public boolean containsKey(ECPublicKey key) {
+		return validators.containsKey(key);
```

Comment:
Two methods the same name, but with different meaning of "key" amused me here.

Reply:
lol